app.js

```diff
@@ -7,4 +7,20 @@ app.use(express.json());
 
 app.get('/api/topics', getTopics);
 
-module.exports = app;
\ No newline at end of file
+app.use('*', (req, res, next) => {
+  next({ status: 404, msg: 'Invalid Path' });
```

Comment:
imstead of passing this onto the next app.use to deal with send the status and msg back from here.